scripts/auto-rebase/rebase.sh

```diff
@@ -221,6 +229,7 @@ clone_repo() {
 
 # Determine the image info for one architecture
 download_image_state() {
+    # shellcheck disable=SC2034
     local release_image="$1"
```

Comment:
This was an unused variable error, the call site is passing a variable over for `$1` what do we want to do here?

Reply:
Let's fix the call to the function and only pass the second variable.

Reply:
Sounds good, updated